introspection-engine/connectors/sql-introspection-connector/src/re_introspection.rs

```diff
@@ -357,7 +364,7 @@ pub fn enrich(old_data_model: &Datamodel, introspection_result: &mut Introspecti
     // potential error: what if there was a db default before and then it got removed, now re-introspection makes it virtual
     // you could not get rid of it
 
-    // println!("{:#?}", new_data_model);
+    println!("{:#?}", new_data_model);
```

Comment:
Was this intentional? Is this part of RPC-over-stdout? If this has side-effects it might be worth sharing that in a comment.

Reply:
Forgot to flag the PR as a draft, this is not going to be in the end result